__main__.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import os
+from multiprocessing import Pool
+
+from bpm.bpm_extractor import get_song_bpm
+from classification.extractor.high_level_data_extractor import make_high_level_data_file
+from classification.classifier.profile_data_extractor import get_classifier_data
+from database.track_bpm import TrackBPM
+from database.track_party import TrackParty
+from database.track_relaxed import TrackRelaxed
+from database.track_timbre import TrackTimbre
+
+
+def extract_and_save_data_from_song(filename):
+    if not filename.endswith(".wav"):
+        print("File is not a .wav file. Exiting...")
+        exit()
+
+    bpm_database_client = TrackBPM()
+    timbre_database_client = TrackTimbre()
+    relaxed_database_client = TrackRelaxed()
+    party_database_client = TrackParty()
+
+    if not os.path.isdir("output/"):
+        os.mkdir("output", 0o755)
+
+    song_id, bpm, bpm_confidence = get_song_bpm(filename)
```

Comment:
I think we should create a separate function to get the song id

Reply:
Agreed, created a Utilities module